product_event_link/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds the event_ids in product.
```

Comment:
```suggestion
This module adds event_ids in product.

The added field was originally part of auction_event module which is private. The field has been moved to this module (LGPL-3) so that the field can be used in AGPL/LGPL modules.
```